modules/apps/dynamic-data-mapping/dynamic-data-mapping-form-field-type/src/main/resources/META-INF/resources/RichText/RichText.es.js

```diff
@@ -271,7 +270,10 @@ const RichText = ({
 						<CKEditor5ClassicEditor
 							className="w-100"
 							config={{
-								...advancedClassicEditorConfig,
+								filebrowserImageBrowseUrl:
+									editorConfig.filebrowserImageBrowseUrl,
```

Comment:
In the future, the idea is to pass entire `config` content. We cannot do it as it is, it would cause a lot of errors, since CKE4 config overrides with incorrect values. We will need to have a separate EditorConfigContributor. We must do this eventually to support customization with EditorConfigContributor, as well as CX. It can be a separate task.